azurerm/data_source_subscription.go

```diff
@@ -0,0 +1,47 @@
+package azurerm
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceArmSubscription() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceArmSubscriptionRead,
+		Schema: map[string]*schema.Schema{
+
+			"subscription_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+
+			"display_name": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
```

Comment:
are there any other fields (e.g. Tenant ID?) that would make sense to be exposed here?

Reply:
I have added the subscription state and possible subscription policies.
I'm not adding the tenant as the [tenant API](https://docs.microsoft.com/en-us/rest/api/resources/tenants) only works for the "current" account.